commitlint.config.cjs

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  'type-enum': [
+    2,
+    'always',
+    ['feat', 'fix', 'docs', 'style', 'refactor', 'perf', 'test', 'build', 'ci', 'chore', 'revert'],
+  ],
+};
```

Comment:
**note**

Set up using `.cjs` extension because of this error: https://github.com/conventional-changelog/commitlint/issues/3251